builtin_interfaces/Time stamp
CamDetection[] detections
